experiment/run_experiment.py

```diff
@@ -128,6 +130,16 @@ def read_and_validate_experiment_config(config_filename: str) -> Dict:
                 'It must start with gs:// when running on Google Cloud.', param,
                 value)
 
+    # Concurrent build can be defined either via parameters or in a config file.
+    # If both define it, they need to be consistent.
+    concurrent_builds_yaml = config.get('concurrent_builds')
+    if concurrent_builds_yaml and concurrent_builds_arg and (
+            concurrent_builds_yaml != concurrent_builds_arg):
+        valid = False
+        logs.error('Inconsistent requirements of number of concurrent builds: '
+                   f'File {config_filename} requests {concurrent_builds_yaml}, '
+                   f'but argument requests {concurrent_builds_arg}.')
+
```

Comment:
Do we also need a check for `worker_pool_name`? For example, should it be in the same zone as `cloud_compute_zone`?

Reply:
No dont bother with that.